doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -233,3 +233,24 @@ The leader network data.
 * Packed-Encoding: `D`
 
 The stable leader network data.
+
+### PROP 5391: PROP_THREAD_JOINERS {#prop-thread-joiners}
+
+* Type: Read-Write
+* Packed-Encoding: `A(T(EDL))`
```

Comment:
My spinel protocol changes have been implemented, so this should now be `A(t(EdL))`.